utils/markdown/renderer.jsx

```diff
@@ -120,19 +119,40 @@ export default class Renderer extends marked.Renderer {
                 }
             }
         }
-        src = PostUtils.getImageSrc(src, this.formattingOptions.proxyImages);
-        let out = '<img src="' + src + '" alt="' + text + '"';
-        if (title) {
-            out += ' title="' + title + '"';
-        }
-        if (dimensions.length > 0) {
-            out += ' width="' + dimensions[0] + '"';
-        }
-        if (dimensions.length > 1) {
-            out += ' height="' + dimensions[1] + '"';
+
+        const regex = /.+\/(.+\.(gifv))(?:\?.*)?$/i;
+        const match = src.match(regex);
+        let out = '';
+
+        //If the URL src is a gifv, then render it in a <video> HTML5 element, otherwise it will go into an <img> element
+        if (match) {
```

Comment:
We'll want to use the image proxy to load these, if possible. Have you tried testing this with the image proxy? If not, we'll want to do that

Reply:
Hmm no, I will look into this as well.

Reply:
I added some tests with basically the hasImageProxy boolean set to true. Is this ok?